localstripe/resources.py

```diff
@@ -1849,16 +1848,23 @@ def status(self):
             return 'requires_payment_method'
         if self.next_action:
             return 'requires_action'
-        if len(self.charges._list) == 0:
+        if self.latest_charge is None:
             return 'requires_confirmation'
-        charge = self.charges._list[-1]
+        charge = self.latest_charge
```

Comment:
This code could be improved by removing line 1853 and using `self.latest_charge` now...